packages/sdk/README.md

```diff
@@ -2,232 +2,65 @@
 
 Using `npm`:
 
 ```bash
-npm install @looker/sdk @looker/sdk-node
+npm install @looker/sdk
 ```
 
 Using `yarn`:
 
 ```bash
-yarn add @looker/sdk @looker/sdk-node
-```
-
-If you are only intending to use the SDK in a browser, omit `@looker/sdk-node`.
-
-### Configure the SDK for your Looker server
-
-**Note**: The `.ini` configuration for the Looker SDK is a sample implementation intended to speed up the initial development of Node applications using the Looker API. See this note on [Securing your SDK Credentials](https://github.com/looker-open-source/sdk-codegen/blob/master/README.md#securing-your-sdk-credentials) for warnings about using `.ini` files that contain your API credentials in a source code repository or production environment.
-
-Create a `looker.ini` file with your server URL and API credentials assigned as shown in this example.
-
-```ini
-[Looker]
-# Base URL for API. Do not include /api/* in the url
-base_url=https://<your-looker-server>:19999
-# API 3 client id
-client_id=your_API3_client_id
-# API 3 client secret
-client_secret=your_API3_client_secret
+yarn add @looker/sdk
 ```
 
-**Note**: If the application using the Looker SDK is going to be committed to a version control system, be sure to
-**ignore** the `looker.ini` file to avoid publishing your API credentials.
-
-### Use the SDK in your code
-
-When the SDK is installed and the server location and API credentials are configured in your `looker.ini` file, it's ready to be used.
+### Authenticate your API calls
 
-Verify authentication works and that API calls will succeed with code similar to the following:
+All requests to the Looker API server require an access token. For browser implementations, authentication is typically achieved via [OAuth](/docs/cors.md) or a [Proxy Server](#using-a-proxy-for-authentication).
 
-```typescript
-import { LookerNodeSDK } from '@looker/sdk-node'
-(async () => {
-  // create a Node SDK object for API 3.1
-  const sdk = LookerNodeSDK.init31()
-  // retrieve your user account to verify correct credentials
-  const me = await sdk.ok(sdk.me(
-    "id, first_name, last_name, display_name, email, personal_space_id, home_space_id, group_ids, role_ids"))
-  console.log({me})
-  // make any other calls to the Looker SDK
-  const dashboards = await sdk.ok(
-    sdk.search_dashboards({title: 'My SDK dashboard'})
-  )
-  if (dashboards.length === 0) {
-    console.log('Dashboard not found')
-  }
-  const [ dashboard ] = dashboards
-  // do stuff with dashboard
+### Use the SDK in your browser application
 
-  ...
+Authenticating for the browser takes a bit more setup than authenticating for use with Node.
 
-  await sdk.authSession.logout()
-  if (!sdk.authSession.isAuthenticated()) {
-    console.log('Logout successful')
-  }
-})()
-```
+The stand-alone version of the Looker API Explorer (included in this repository) uses OAuth and the `BrowserSDK` to get an authentication token for Looker API requests.
```

Comment:
Can we provide an outline of what needs to be done

1. register before starting
2. get the 2 urls
3. auth with redirect back
4. hopefully logged in

Reply:
That's described in `cors.md` but probably that doc could use an update and this section could duplicate it.